activerecord/test/cases/signed_id_test.rb

```diff
@@ -178,11 +178,29 @@ def set_signed_id
     ActiveRecord::Base.signed_id_verifier_secret = SIGNED_ID_VERIFIER_TEST_SECRET
   end
 
-  test "always output url_safe" do
+  test "always output url_safe when url_safe_signed_id is true" do
+    Account.url_safe_signed_id = true
```

Comment:
This is leaking global state and might leak to flaky tests in the future.

Either every test that set `Account.url_safe_signed_id` should restore the previous value, or it should be reset in `teardown`.

Reply:
Good point!
I restored the previous state at the end of these two tests for `url_safe_signed_id`.